example/dev.auto.tfvars

```diff
@@ -1,16 +1,15 @@
 region                            = "us-east-1"
-environment                       = "dev"
-profile                           = "default"
-namespace                         = "refarchdevops"
-route_53_zone                     = "sfrefarch.com"
+environment                       = "poc"
+namespace                         = "arc"
+route_53_zone                     = "sfarcpoc.com"
 name                              = "sl-k8s"
-kubernetes_version                = "1.21"
+kubernetes_version                = "1.25"
```

Comment:
Can we try with `1.28` version? After all `1.29 `is the latest one i guess

Reply:
yeah , will be testing it